packages/cactus-test-plugin-htlc-eth-besu-erc20/package.json

```diff
@@ -61,7 +61,7 @@
     "axios": "1.6.0",
     "express": "4.19.2",
     "web3-eth-abi": "4.0.3",
-    "web3-utils": "4.2.1"
+    "web3-utils": "1.7.0"
```

Comment:
There we go again with the downgrades that are unexplained. 
cc: @hyperledger/cacti-maintainers 

I'm worried about dependabot.

Reply:
I will have a look into projects using dependabot which are massive (with a lot of depedencies at multiple levels) to see how do they deal with such inconsistencies. Is it mandatory for us to use this bot, @petermetz ? Because if that's not the case, we can have a periodic trivy scan for dependency upgrades based on vulnerabilities. (And anyway the bot just vaguely updates the dependencies which creates another set of problems many a times)